addons/website_slides/controllers/main.py

```diff
@@ -618,6 +583,51 @@ def _slide_channel_prepare_values(self, **kw):
             'allow_comment': bool(kw.get('allow_comment')),
         }
 
+    def _slide_channel_prepare_review_values(self, channel):
+        values = {
+            'channel': channel,
```

Comment:
Channel is already present in call done in ``channel`` route method, and as this method is about rating values I would not add anything else (caller's job).